File: backend/src/main/java/com/frh/backend/dto/ListingReviewResponse.java
package com.frh.backend.dto;

import java.time.LocalDateTime;
import lombok.Data;

@Data
public class ListingReviewResponse {

  private Long reviewId;
  private Long orderId;
  private Long listingId;
  private String listingTitle;
  private Integer storeRating;
  private Integer listingAccuracy;
  private Integer onTimePickup;
  private String comment;
  private LocalDateTime createdAt;
  private Long consumerId;
  private String consumerDisplayName;
}

File: android/app/src/main/java/com/example/foodrescuehub/data/model/ListingReviewResponse.kt
package com.example.foodrescuehub.data.model

data class ListingReviewResponse(
    val reviewId: Long,
    val orderId: Long,
    val listingId: Long,
    val listingTitle: String?,
    val storeRating: Int,
    val listingAccuracy: Int,
    val onTimePickup: Int,
    val comment: String,
    val createdAt: String,
    val consumerId: Long,
    val consumerDisplayName: String
)
